test/fuzz/corpus/jwt_verify_lib_fuzz_test/jwks_ec.txt

```diff
@@ -0,0 +1,3 @@
+jwt: "base64ed+header.base64+claims.base64+signature"
+jwks: '{"keys":[{"kty":"EC","crv":"P-256","alg":"ES256","kid":"abc","x":"EB54wykhS7YJFD6RYJNnwbWEz3cI7CF5bCDTXlrwI5k","y":"92bCBTvMFQ8lKbS2MbgjT3YfmYo6HnPEE2tsAqWUJw8"},{"kty":"EC","crv":"P-256","alg":"ES256","kid":"xyz","x":"EB54wykhS7YJFD6RYJNnwbWEz3cI7CF5bCDTXlrwI5k","y":"92bCBTvMFQ8lKbS2MbgjT3YfmYo6HnPEE2tsAqWUJw8"},{"kty":"EC","crv":"P-384","alg":"ES384","kid":"es384","x":"yY8DWcyWlrr93FTrscI5Ydz2NC7emfoKYHJLX2dr3cSgfw0GuxAkuQ5nBMJmVV5g","y":"An5wVxEfksDOa_zvSHHGkeYJUfl8y11wYkOlFjBt9pOCw5-RlfZgPOa3pbmUquxZ"},{"kty":"EC","crv":"P-521","alg":"ES512","kid":"es512","x":"Abijiex7rz7t-_Zj_E6Oo0OXe9C_-MCSD-OWio15ATQGjH9WpbWjN62ZqrrU_nwJiqqwx6ZsYKhUc_J3PRaMbdVC","y":"FxaljCIuoVEA7PJIaDPJ5ePXtZ0hkinT1B_bQ91mShCiR_43Whsn1P7Gz30WEnLuJs1SGVz1oT4lIRUYni2OfIk"}]}'
```

Comment:
Based on https://developers.google.com/protocol-buffers/docs/text-format-spec#string, I don't think single quote in textformat allows you to use unescaped `"` or `\n` characters.

Suggest you escape the string (you can use "moma/cyberchef").

Reply:
I just added a test to test all corpus file formats.